packages/frontend/src/hooks/useAllTimeStats.tsx

```diff
@@ -0,0 +1,41 @@
+import { useQuery } from '@tanstack/react-query';
+import { request, gql } from 'graphql-request';
+import GRAPH_URLS from 'src/subgraph.json';
+
+const ALL_TIME_STATS_QUERY = gql`
+  query {
+    allTimeStats(id: "current") {
+      volumeUsd
+    }
+  }
+`;
+
+type AllTimeStatsResponse = {
+  allTimeStats: {
+    volumeUsd: string;
+  };
+};
+
+const useAllTimeStats = () => {
+  return useQuery(['allTimeStats'], async () => {
+    const responses = (await Promise.all(
```

Comment:
Would be nice to be explict here about when data should be refetched.

Currently I think the data is immediately being marked "stale" - which we should be able to control with `staleTime`. We also have [refetch on window focus](https://tanstack.com/query/v4/docs/react/guides/window-focus-refetching) for this query - do we want this?

Reply:
Great call, addressed in [f381bb6](https://github.com/sifiorg/sifi/pull/362/commits/f381bb62cd63b951f0a4d48c7ea3aec799b85717).